tiny_dnn/core/framework/tensor.h

```diff
@@ -172,6 +172,42 @@ class Tensor {
   }
 
   /**
+   * Access to indexes in tensor with or mixed vector/indices
+   * @param vec first indices
+   * @param args rest of indices
+   * @return the value of a specified index in the tensor
+   */
+  template <typename... Args>
+  U host_at(std::vector<size_t> vec, const Args... args) const {
+    std::vector<size_t> rest = {args...};
+    vec.insert(vec.end(), rest.begin(), rest.end());
+    return host_at(vec);
+  }
+
+  /**
+   * Access to indexes in tensor with vector
+   * @param vec vector with first indices
+   * @return the value of a specified index in the tensor
+   */
+  U host_at(std::vector<size_t> vec) const {
```

Comment:
Here it might be a constant reference, in previous version I changed the vector, so I didn't want to use a reference